packages/core/content-manager/server/routes/admin.js

```diff
@@ -388,7 +388,22 @@ module.exports = {
     {
       method: 'GET',
       path: '/collection-types/:model/:id/actions/numberOfDraftRelations',
-      handler: 'collection-types.getNumberOfDraftRelations',
+      handler: 'collection-types.countDraftRelations',
+      config: {
+        middlewares: [routing],
+        policies: [
+          'admin::isAuthenticatedAdmin',
+          {
+            name: 'plugin::content-manager.hasPermissions',
+            config: { actions: ['plugin::content-manager.explorer.read'] },
+          },
+        ],
+      },
+    },
+    {
+      method: 'GET',
+      path: '/collection-types/:model/actions/multipleEntriesNumberOfDraftRelations',
```

Comment:
shouldn't we update the path too?

Reply:
ok sure thanks

Reply:
Do you think it is safe also to change the other endpoint `numberOfDraftRelations` to `countDraftRelations`?

Reply:
done